views/account/signup.pug

```diff
@@ -8,15 +8,15 @@ block content
     .form-group.row
       label.col-md-3.col-form-label.font-weight-bold.text-right(for='email') Email
       .col-md-7
-        input.form-control(type='email', name='email', id='email', placeholder='Email', autofocus, required)
+        input.form-control(type='email', name='email', id='email', placeholder='Email', autofocus, autocomplete="email", required)
     .form-group.row
       label.col-md-3.col-form-label.font-weight-bold.text-right(for='password') Password
       .col-md-7
-        input.form-control(type='password', name='password', id='password', placeholder='Password', required)
+        input.form-control(type='password', name='password', id='password', placeholder='Password', autocomplete="existing-password", required)
```

Comment:
Since this is when we are creating an account, I think new-password is better fitting per the description in the specs.

> A new password (e.g. when creating an account or changing a password)

Reply:
good catch, changed